src/fidesops/graph/graph.py

```diff
@@ -11,11 +11,10 @@
     SeedAddress,
     Collection,
     Dataset,
-    EdgeDirection,
     Field,
     FieldPath,
 )
-from fidesops.schemas.shared_schemas import FidesOpsKey
+from fidesops.schemas.shared_schemas import EdgeDirection, FidesOpsKey
```

Comment:
Moved FidesOpsKey and EdgeDirection into shared_schemas to avoid a circular dependency.